js/components/portal-dashboard/feedback/activity-level-feedback-student-rows.tsx

```diff
@@ -24,67 +25,69 @@ interface IProps {
 
-    return (
-      <div key={activityId + studentId} className={css.feedbackRowsRow} data-cy="feedbackRow">
-        <div className={css.studentWrapper}>
-          <div className={css.feedbackBadge} data-cy="feedback-badge">
-            { activityStarted && feedbackBadge }
-          </div>
-          <div className={css.studentName} data-cy="student-name">
-            {formattedName}
+      return (
+        <div key={activityId + studentId} className={css.feedbackRowsRow} data-cy="feedbackRow">
+          <div className={css.studentWrapper}>
+            <div className={css.feedbackBadge} data-cy="feedback-badge">
+              {activityStarted && feedbackBadge}
+            </div>
+            <div className={css.studentName} data-cy="student-name">
+              {formattedName}
+            </div>
           </div>
-        </div>
-        <div className={css.feedback} data-cy="feedback-container">
-          {activityStarted && hasRubric &&
-            <RubricTableContainer
+          <div className={css.feedback} data-cy="feedback-container">
+            {activityStarted && hasRubric &&
+              <RubricTableContainer
+                activityId={activityId}
+                activityIndex={activityIndex}
+                rubric={rubric}
+                student={student}
+                rubricFeedback={rubricFeedback}
+                setFeedbackSortRefreshEnabled={setFeedbackSortRefreshEnabled}
+                updateActivityFeedback={updateActivityFeedback}
+              />
+            }
+            <ActivityFeedbackTextarea
               activityId={activityId}
               activityIndex={activityIndex}
-              rubric={rubric}
-              student={student}
-              rubricFeedback={rubricFeedback}
+              activityStarted={activityStarted}
+              feedback={feedback}
+              key={activityId + studentId + "-textarea"}
+              studentId={studentId}
               setFeedbackSortRefreshEnabled={setFeedbackSortRefreshEnabled}
               updateActivityFeedback={updateActivityFeedback}
+              trackEvent={trackEvent}
             />
-          }
-          <ActivityFeedbackTextarea
-            activityId={activityId}
-            activityIndex={activityIndex}
-            activityStarted={activityStarted}
-            feedback={feedback}
-            key={activityId + studentId + "-textarea"}
-            studentId={studentId}
-            setFeedbackSortRefreshEnabled={setFeedbackSortRefreshEnabled}
-            updateActivityFeedback={updateActivityFeedback}
-            trackEvent={trackEvent}
-          />
+          </div>
         </div>
+      );
+    });
+
+    return (
+      <div className={css.feedbackRows}>
+        {feedbackRows}
       </div>
     );
-  });
-
-  return (
-    <div className={css.feedbackRows}>
-      {feedbackRows}
-    </div>
-  );
-};
+  }
+}
```

Comment:
I'm confused why this was changed to a class.  I think we want to keep it as a functional component.  Unless there is some specific advantage to having it a class.  Moving forward I think we want to use functional components whenever possible since React is moving in that direction.